README.md

```diff
@@ -0,0 +1,177 @@
+# Apache Airflow DAG with building permissions - project
+
+## Why did I create this project?
+
+This project is a result of my learning and practicing about DAGs in Apache Airflow and visualisations made in Jupyter Notebook or Looker Studio.
+
+## What does this project do?
+
+This project uses Apache Airflow's directed acyclic graph (or, shortly - DAG) to organize work with data coming from polish governmental Main Building Supervision Office. The data is related to building permissions.
+
+The DAG works with the following tasks (in order):
+
+1. **zip_data_downloader_task** - this task is responsible for downloading the building permissions data and unzipping it (to the original .csv format).
+2. **validation task** - this task does the validation of the selected columns of the downloaded data using Great Expectations library. There are four goals of the validation which are: 
+    - checking if the dates in 'data_wplywu_wniosku_do_urzedu' column have the correct format
+    - checking if the values in 'kategoria' column are the expected numbers of the roman type
+    - checking if the 'terc' column includes the correct type (7 digits) of TERC codes
+    - checking if the 'rodzaj_zam_budowlanego' column includes all 4 expected types of building construction intention.
+3. **unzipped_data_uploader_task** - this task uploades the validated data (a table named "reporting_results2020") to a BigQuery database.
+4. **aggregates_creation_task** - this task uses the validated data to create a customized aggregates. The aggregates table consists of columns including information about how many building actions were reported to the polish Main Building Supervision Office in the last 3, 2 and 1 month (divided also by building categories types). At the end, the task sends the aggregates table to the BigQuery database.
+5. **send_email_task** - this task sends a reporting email to a certain mail address. It has the validation report from "validation task" (in .html format).
+
+This DAG is scheduled to be executed once per month, on a first day of each month. It does catchups if necessary.
+
+Moreover, for this project, I created visualisations basing on the "reporting_results2020" table in Jupyter Notebook. The visualisations include various plots and GIF file showing the changes in the number of building permissions per last 3, 2 and 1 month (the GIF is below)
```

Comment:
Nice plot, but it will be good to change the color pallete to nicer colors (of course this is my personal suggestion), and also add the date to the graph. Other thing is that showing this plot in time like from 2023.01 to 2023.12 could give us much better perspective but similat thing you will made in looker :)